### DISCUSSION TITLE
Welcome to poco Discussions!

### DISCUSSION THREAD

**Discussion by obiltschnig** [pocoproject/poco#3164] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zosrothko** [comment]
Well, as first commenter, welcome everybody ! Let's make Poco the universal tool for building industrial strength C++ applications.
Thanks to @obiltschnig & @aleks-f  for this awesome project!

I am a Poco fanatic focusing the development of C++ applications with Poco mainly on VisualStudio. 


> In Code We Trust!

**Comment by Borwe** [comment]
Hmmmm. I read somewhere on reddit pocco was dead. Is it true?

**Comment by obiltschnig** [comment]
Well, the reports of Poco's death have been greatly exaggerated.

Development has slowed down quite a bit this year for various reasons, but we plan a new release for early next year.

**Reply by Borwe** [reply]
> 
> 
> Well, the reports of Poco's death have been greatly exaggerated.
> 
> Development has slowed down quite a bit this year for various reasons, but we plan a new release for early next year.

Nice.
